### DISCUSSION TITLE
awesome-flet?

### DISCUSSION THREAD

**Discussion by xzripper** [flet-dev/flet#2057] [General]
There are many impressive community projects that enhance flet with new widgets, utilities, and more. Why not create an `awesome-flet` repository where users can add and contribute their own projects? It would be advantageous and convenient to compile a comprehensive list of all projects created by members of the community. Repository can be hosted by community, or by flet developers team.

**Comment by ndonkoHenri** [comment]
Was thinking of that lately when I saw awesomeflutter and python. Will probably dive more into that this weekend.

**Comment by FeodorFitsner** [comment]
Hey guys, I can create `flet-dev/awesome-flet` repo and give you rights to manage it. Would you like to manage it? 😎

**Comment by xzripper** [comment]
I'd like to do it!😁

**Comment by FeodorFitsner** [comment]
Awesome! :)
Repo created, invites sent: https://github.com/flet-dev/awesome-flet

**Comment by xzripper** [comment]
Accepted :)
